the-collab-lab/tcl-45-smart-shopping-list
# Mk list item style

## Description

This PR styles the `ListItem` aka the items that come up in each list according to its time frame. I used flex box to align the items in columns under the titles of "Item", "Bought" and "Delete", and put the corresponding ghost colors for the time frames that were used in `AddItem` in the title of each section. 

## Related Issue

Closes #63 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |
|  ✓   | 💅 : Styling              |

## Updates

### Before
![Screen Shot 2022-08-27 at 9 00 43 PM](https://user-images.githubusercontent.com/90009901/187056750-1685cd27-3f7c-4324-96c9-e76b5a667610.png)


### After

![Screen Shot 2022-08-27 at 8 55 08 PM](https://user-images.githubusercontent.com/90009901/187056739-3e93b6a6-672d-40d6-ad15-56cf1ff331c5.png)


## Testing Steps / QA Criteria
1. git checkout main then git pull
2. git checkout MN-list-item-style
3. npm run start
4. Navigate to your List page and checkout list items
5. add different list items with different time frames to check for organization and responsiveness
